src/components/global-header/global-header.stories.mdx

```diff
@@ -0,0 +1,225 @@
+              <MenuItem href="#">Child Item 3</MenuItem>
+            </MenuItem>
+            <MenuItem flex="0 0 auto" submenu="Parent Menu 2">
+              <MenuItem>Child Item</MenuItem>
+            </MenuItem>
+          </Menu>
+        </GlobalHeader>
+      );
+    }}
+  </Story>
+</Canvas>
+
+### Responsive menu
+
+This story is best viewed in the `canvas` view and by adjusting the size of the window. The fullscreen behaviour will trigger when the screen size is smaller than `600px`.
+
+<Canvas>
+  <Story name="responsive menu">
+    {() => {
+      const Logo = () => <img height={28} src={carbonLogo} alt="Carbon logo" />;
+      const fullscreenViewBreakPoint = useMediaQuery("(max-width: 599px)");
+      const [isListViewOpen, setIsListViewOpen] = useState(false);
+      const menuItems = [
+        <MenuItem
+          key="product-switcher"
+          minWidth="160px"
+          flex="1"
+          submenu="Product Switcher"
+        >
+          <MenuItem>Product A</MenuItem>
+        </MenuItem>,
+        <MenuItem
+          key="parent-menu-1"
+          minWidth="145px"
+          flex="0 0 auto"
+          submenu="Parent Menu 1"
+        >
+          <MenuItem href="#">Child Item 1</MenuItem>
+          <MenuSegmentTitle>Segment title</MenuSegmentTitle>
+          <MenuItem href="#">Child Item 2</MenuItem>
+          <MenuItem href="#">Child Item 3</MenuItem>
+        </MenuItem>,
+        <MenuItem
+          key="parent-menu-2"
+          minWidth="145px"
+          flex="0 0 auto"
+          submenu="Parent Menu 2"
+        >
+          <MenuItem>Child Item</MenuItem>
+        </MenuItem>,
+      ];
+      return (
+        <GlobalHeader logo={<Logo />}>
+          <Menu menuType="black" display="flex" flex="1">
+            {fullscreenViewBreakPoint ? (
+              <>
+                <MenuItem
+                  onClick={(ev) => {
+                    ev.preventDefault();
+                    setIsListViewOpen(true);
+                  }}
+                >
+                  Admin
+                </MenuItem>
+                <MenuFullscreen
+                  menuType="black"
+                  isOpen={isListViewOpen}
+                  onClose={() => setIsListViewOpen(false)}
+                >
+                  {menuItems}
+                </MenuFullscreen>
+              </>
+            ) : (
+              menuItems
+            )}
+          </Menu>
+        </GlobalHeader>
+      );
+    }}
+  </Story>
+</Canvas>
+
+### Global and local navigation bar layout
+
+The component can be used alongside the <LinkTo kind='Navigation Bar' story="default">Navigation Bar component</LinkTo> to create a two navigation bar layout, with the former being for site-wide navigation and the latter being for local navigation. Since Global Header is fixed to top of the viewport, the Navigation Bar must also have its `position` fixed, with a top `orientation` and `offset` of 40px.
```

Comment:
```suggestion
The component can be used alongside the <LinkTo kind='Navigation Bar' story="default">NavigationBar component</LinkTo> to create a two navigation bar layout, with the former being for site-wide navigation and the latter being for local navigation. Since GlobalHeader is fixed to top of the viewport, the NavigationBar must also have its `position` fixed, with a top `orientation` and `offset` of 40px.
```

Same comment as referenced a couple of times already.